contracts/global-extensions/GlobalOptimisticAuctionRebalanceExtension.sol

```diff
@@ -0,0 +1,328 @@
+/*
+    Copyright 2023 Index Coop
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+    SPDX-License-Identifier: Apache License, Version 2.0
+*/
+
+pragma solidity 0.6.10;
+pragma experimental "ABIEncoderV2";
+
+import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import {SafeERC20} from  "@openzeppelin/contracts/token/ERC20/SafeERC20.sol";
+
+import { AddressArrayUtils } from "../lib/AddressArrayUtils.sol";
+
+import { BaseGlobalExtension } from "../lib/BaseGlobalExtension.sol";
+import { IAuctionRebalanceModuleV1 } from "../interfaces/IAuctionRebalanceModuleV1.sol";
+import { IManagerCore } from "../interfaces/IManagerCore.sol";
+import { ISetToken } from "../interfaces/ISetToken.sol";
+import { IDelegatedManager } from "../interfaces/IDelegatedManager.sol";
+import {GlobalAuctionRebalanceExtension} from "./GlobalAuctionRebalanceExtension.sol";
+import {AncillaryData } from "../lib/AncillaryData.sol";
+import {OptimisticOracleV3Interface} from "../interfaces/OptimisticOracleV3Interface.sol";
+
+/**
+ * @title GlobalOptimisticAuctionRebalanceExtension
+ * @author Index Coop
+ *
+ * @dev Extension contract for interacting with the AuctionRebalanceModuleV1. This contract acts as a pass-through and functions 
+ * are only callable by operator. 
+ */
+contract GlobalOptimisticAuctionRebalanceExtension is  GlobalAuctionRebalanceExtension {
```

Comment:
should we inherit `OptimisticOracleV3CallbackRecipientInterface` ?https://github.com/UMAprotocol/protocol/blob/cef7d8880a2168eeb30f54151a8d40b274fbd381/packages/core/contracts/optimistic-governor/implementation/OptimisticGovernor.sol#L28C43-L28C43

Reply:
wouldn't be as simple as just implementing the callbacks and running integration tests, but I think it is more proper while the former would also be good enough for an extension. I'd hope to be able to just have uma as dependency but working with the different compilers could quickly evolve to a lot of rewriting and even more glue for this repo.